uniffi_bindgen/src/bindings/kotlin/templates/NamespaceLibraryTemplate.kt

```diff
@@ -18,13 +18,13 @@ inline fun <reified Lib : Library> loadIndirect(
 
 internal interface _UniFFILib : Library {
     companion object {
-        internal val INSTANCE: _UniFFILib by lazy { 
+        internal val INSTANCE: _UniFFILib by lazy {
             loadIndirect<_UniFFILib>(componentName = "{{ ci.namespace() }}")
-            {% let callback_interfaces = ci.iter_callback_interface_definitions() %}
-            {%- if !callback_interfaces.is_empty() -%}
+            {% let initialization_code = self.initialization_code() %}
+            {%- if !initialization_code.is_empty() -%}
             .also { lib: _UniFFILib ->
-                {% for cb in callback_interfaces -%}
-                CallbackInterface{{ cb.name()|class_name_kt }}Internals.register(lib)
+                {% for code in initialization_code -%}
+                {{ code }}
```

Comment:
Initialization is now distributed to the edges (i.e. the members `CodeDeclaration`s). Currently, only `callback_interface.rs` uses it.